packages/jest-reporters/src/coverage_reporter.ts

```diff
@@ -45,15 +45,15 @@ export default class CoverageReporter extends BaseReporter {
     this._options = options || {};
   }
 
-  onTestResult(
-    _test: Test,
-    testResult: TestResult,
-    _aggregatedResults: AggregatedResult,
-  ) {
+  onTestResult(_test: Test, testResult: TestResult) {
     if (testResult.coverage) {
       this._coverageMap.merge(testResult.coverage);
     }
 
+    if (this._globalConfig.v8Coverage) {
```

Comment:
no need to deal with sourcemaps. I'm not happy with this implementation, we might wanna move the `v8ToIstanbul` logic into this reporter instead of doing it in `jest-runtime`

Reply:
this has been done